package.json

```diff
@@ -2,12 +2,13 @@
   "name": "@xmtp/proto",
   "version": "0.0.0-development",
   "description": "Protobuf client and generated classes for GRPC API",
-  "main": "./ts/dist/cjs/index.js",
-  "module": "./ts/dist/esm/index.js",
+  "main": "./ts/dist/cjs/node.js",
+  "module": "./ts/dist/esm/node.js",
   "types": "./ts/dist/types/index.d.ts",
+  "browser": "./ts/dist/esm/index.js",
```

Comment:
This is how I conditionally exclude the polyfill package entirely from browser builds

Reply:
Is there an advantage to using the browser-provided fetch? I would almost lean to using the polyfill for everything for consistency / making it easier to debug since we have to support it anyways

Reply:
Undici only works in browsers with some complicated WASM builds, since it's full of native code. It'll also blow up the bundle size (it's quite large).